docs/pages/1-generate-https.md

```diff
@@ -168,205 +166,64 @@ this is pronounced ENGINE-X - it took me only 8 years to know that :P). As a san
-sudo mv acme-tiny /opt/acme-tiny/
-sudo chown $USER -R /opt/acme-tiny
+sudo add-apt-repository ppa:certbot/certbot
+sudo apt-get update
+sudo apt-get install python-certbot-nginx
 ```
 
-### Step 3. Create Account Key and Parameters
-This step you only need to do once, so I'm bundling it into one. You need to generate a 
-key for your server, and for the best encryption, a `dhparam.pem` key (it takes a while!).
+### Step 2. Get certificates with certbot
 
-```bash
-# Generate a private account key, if doesn't exist
-if [ ! -f "/etc/ssl/certs/account.key" ]
-   then
-   openssl genrsa 4096 > account.key && sudo mv account.key /etc/ssl/certs
-fi
+Now obtain a certificate by running this command. 
+You can specify multiple hostnames, such as one with "www" like this:
 
-# Add extra security (takes a while!)
-if [ ! -f "/etc/ssl/certs/dhparam.pem" ]
-   then
-   openssl dhparam -out dhparam.pem 4096 && sudo mv dhparam.pem /etc/ssl/certs
-fi
 ```
-
-You shouldn't need to regenerate these files.
-
-### Step 4. Backup Previous Key / Certificate
-This step you only need to do if you have a previously created certificate and domain
-key. It's probably not super necessary, but I do it anyway. The certificate we will make
-(and/or backup) is called `expfactory.cert` and the key is `expfactory.key`.
-
-```bash
-# backup old key and cert
-if [ -f "/etc/ssl/private/domain.key" ]
-   then
-   sudo cp /etc/ssl/private/domain.key{,.bak.$(date +%s)}
-fi
-
-if [ -f "/etc/ssl/certs/chained.pem" ]
-   then
-   sudo cp /etc/ssl/certs/chained.pem{,.bak.$(date +%s)}
-fi
-
-if [ -f "/etc/ssl/certs/domain.csr" ]
-   then
-   sudo cp /etc/ssl/certs/domain.csr{,.bak.$(date +%s)}
-fi
+certbot certonly --nginx -d expfactory.dynu.net -d www.expfactory.dynu.net
 ```
 
-### Step 5. Call Openssl
-We now are going to use openssl with our `csr_details.txt` to generate a new domain key! That looks like this:
+This will save all of the certificate files to /etc/letsencrypt/live/expfactory.dynu.net (the first domain listed in the cerbot command). It will ask for your email during the generation. This is used to send you renewal reminders at 20 days, 10 days, and 1 day before expiry (super helpful!)
 
-```bash
-openssl req -new -sha256 -nodes -out domain.csr -newkey rsa:2048 -keyout domain.key -config <( cat csr_details.txt )
+### Step 3. Copy certs to a new location
 
-# Move to where they are expected by the container
-sudo mv domain.csr /etc/ssl/certs/domain.csr
-sudo mv domain.key /etc/ssl/private/domain.key
-```
+Now we'll move these to where they're expected later.
 
-### Step 6. The Acme Challenge!
-Remember Acme Tiny? Let's use it now. Acme Tiny is going to help us communicate with Let's Encrypt. First, create the challenge folder in the webroot of your local nginx:
+First set the hostname for your server as a shell variable:
 
-```bash
-sudo mkdir -p /var/www/html/.well-known/acme-challenge/
-sudo chown $USER -R /var/www/html/
+```
+MYDOMAIN="expfactory.dynu.net"
```

Comment:
Ah! Here is where the defined variable is handy, because they wouldn't need to define it again here.